### PR TITLE
Install all runtime dependencies from binary packages

### PR DESCRIPTION
**Pull request by mattias-p**
With this PR we're satisfying all of Engine's runtime dependencies with OS packages, fixing #451.

* Text::CSV is taken from EPEL on CentOS,
* Mail::Address::RFC822 is replaced with Email::Valid, and
* the minimum required version for Locale::TextDomain is lowered to 1.20.

Note that this is for 2019.2.

### CODE REVIEW

File: Makefile.PL [hunk 1]
```diff
@@ -16,20 +16,20 @@ all_from 'lib/Zonemaster/Engine.pm';
...
-requires 'Text::CSV'             => 0;
-requires 'Zonemaster::LDNS'      => 2.0;
+requires 'Clone'              => 0;
+requires 'Email::Valid'       => 0;
+requires 'File::ShareDir'     => 1.00;
+requires 'File::Slurp'        => 0;
+requires 'IO::Socket::INET6'  => 2.69;
+requires 'JSON::PP'           => 0;
+requires 'List::MoreUtils'    => 0;
+requires 'Locale::TextDomain' => 1.20;
```
Comment (matsduf) [inline Makefile.PL]: We have previously concluded that we need version 1.23 of Locale::TextDomain for translation to work in all environments.

**Comment by mattias-p** [comment]
I'll test Debian, @matsduf has promised to test on FreeBSD, @sandoche2k has promised to test on CentOS and Ubuntu.

**Comment by matsduf** [comment]
Issue for FreeBSD in #556. Waiting for this to be merged.

**Comment by mattias-p** [comment]
I tested this on Debian 8 and Debian 9 and it works on both.

**Pull request merged**